src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -212,8 +214,9 @@ namespace service_nodes
     service_nodes::quorum_type const max_quorum_type = service_nodes::max_quorum_type_for_hf(hf_version);
     bool tested_myself_once_per_block                = false;
 
-    time_t const now         = time(nullptr);
-    uint64_t const live_time = (now - m_core.get_start_time());
+    ptrdiff_t start_time      = m_core.get_start_time();
```

Comment:
A `ptrdiff_t` is not guaranteed to be big enough to hold a time value, which could eventually be a problem here (versus using a time_t).

Reply:
Fixed